src/api/firebase.js

```diff
@@ -70,7 +74,28 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 	});
 }
 
-export async function updateItem() {
+export async function updateItem(listId, itemId, checked) {
+	const listItemRef = doc(db, listId, itemId);
+	const listItemSnap = await getDoc(listItemRef);
+	const currentTotalPurchases = listItemSnap.data().totalPurchases;
+	const currentTime = listItemSnap.data().dateLastPurchased;
+
+	let totalPurchases = currentTotalPurchases;
+	let dateLastPurchased = currentTime;
+	let elapsedTime = (new Date() - currentTime) / (1000 * 60 * 60 * 24);
```

Comment:
I mentioned this in the comment below, but `elapsedTime` may not be what you expect here.

Reply:
Haha it definitely was not. That's where we were getting that weird value in `dateLastPurchased` from. Ended up taking that out for now.